src/main/java/org/manifold/compiler/front/Main.java

```diff
@@ -195,8 +193,43 @@ public Schematic invokeFrontend(CommandLine cmd) throws Exception {
     // Pass the tokens to the parser
     ManifoldParser parser = new ManifoldParser(tokens);
 
+    StringBuilder errors = new StringBuilder();
```

Comment:
Oh, cool -- can you paste me a sample of the output from this when there are parser errors?